packages/component-submission/server/use-cases/submitSurveyUseCase.test.js

```diff
@@ -0,0 +1,108 @@
+const { useCase, makeResponseObject } = require('./submitSurveyUseCase')
+
+describe('submitSurveyUseCase', () => {
+  describe('useCase', () => {
+    it('does not supress errors', () => {
+      class MockSurveyResponseThrowsOnSave {
+        constructor(_) {
```

Comment:
You can avoid needing empty constructors by using object literals. Best practice in jest is to wrap a function that returns one in a `jest.fn` 

```
const MockSurveyResponseEntity = jest.fn(() => ({
  save: jest.fn(() => throw new Error('error'))
}))
```

Reply:
Fixed - I also discovered that you're meant to return async expects - who knew :man_shrugging: